tests/Controller/CRUDControllerTest.php

```diff
@@ -426,12 +395,12 @@ public function testConfigureWithException(): void
 
     public function testConfigureWithException2(): void
     {
+        $this->expectException(\RuntimeException::class);
```

Comment:
Why are you moving these expectations here?

Reply:
Expect Exception in comment is deprecated (https://phpunit.readthedocs.io/en/latest/writing-tests-for-phpunit.html#testing-exceptions)

Reply:
I'm not proposing to use comments. I'm asking why are we moving the calls to `expectException()` and `expectExceptionMessage()` previous the lines they were originally.